src/core/binding.c

```diff
@@ -1694,7 +1674,7 @@ QuicBindingReceive(
         // Perform initial validation.
         //
         BOOLEAN ReleaseDatagram;
-        if (!QuicBindingPreprocessDatagram(Binding, Datagram, &ReleaseDatagram)) {
+        if (!QuicBindingPreprocessDatagram(Binding, (QUIC_RX_PACKET*)Datagram, &ReleaseDatagram)) {
```

Comment:
Should this function be renamed to QuicBindingPreprocessPacket?

Reply:
Yes